lib/aws/AwsSigv4Signer.js

```diff
@@ -15,9 +15,10 @@ const Transport = require('../Transport');
 const aws4 = require('aws4');
 const AwsSigv4SignerError = require('./errors');
 
-async function AwsSigv4Signer(opts) {
+function AwsSigv4Signer(opts) {
   const credentialsState = {
     credentials: null,
+    acquiredAt: 0,
```

Comment:
Why do we need `acquiredAt` here? I don't see it being used anywhere to do anything.

Reply:
yep, I was in two mind about it, it's something I kept around thinking it could be useful to have it for debugging purposes - But since it has no use at the moment, let's remove it.